src/HDF5Msg.chpl

```diff
@@ -1782,47 +1774,31 @@ module HDF5Msg {
      */
     proc readAllHdfMsg(cmd: string, payload: string, st: borrowed SymTab): MsgTuple throws {
         var repMsg: string;
-        // May need a more robust delimiter then " | "
-        var (strictFlag, ndsetsStr, nfilesStr, allowErrorsFlag, calcStringOffsetsFlag, arraysStr) = payload.splitMsgToTuple(6);
-        var strictTypes: bool = true;
-        if (strictFlag.toLower().strip() == "false") {
-          strictTypes = false;
-        }
+        var msgArgs = parseMessageArgs(payload, 7);
+        var strictTypes: bool = msgArgs.get("strict_types").getBoolValue();
 
-        var allowErrors: bool = "true" == allowErrorsFlag.toLower(); // default is false
+        var allowErrors: bool = msgArgs.get("allow_errors").getBoolValue(); // default if false
```

Comment:
this is not important but I think the comment was correct before

Reply:
Yeah, I think I removed it and fat-fingered it when I added it back. Fixed now.